sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -122,7 +126,7 @@ statement
     | ALTER (TABLE | VIEW) tableIdentifier
         UNSET TBLPROPERTIES (IF EXISTS)? tablePropertyList             #unsetTableProperties
     | ALTER TABLE tableIdentifier partitionSpec?
-        CHANGE COLUMN? identifier colType colPosition?                 #changeColumn
+        CHANGE COLUMN? errorCapturingIdentifier colType colPosition?   #changeColumn
```

Comment:
Also, in this case, the current error handling and the output message is short and succinct enough. Let's not touch `ALTER TABLE` statements.
```
scala> sql("ALTER TABLE t-c CHANGE COLUMNS a (a int)").show
org.apache.spark.sql.catalyst.parser.ParseException:
no viable alternative at input 'ALTER TABLE t-'(line 1, pos 13)

== SQL ==
ALTER TABLE t-c CHANGE COLUMNS a (a int)
-------------^^^
```

Reply:
To the Spark SQL expert like us, this error message is very simple to understand. :-) However, that might not be true for our end users, who might not understand this.